cmd/kpod/create.go

```diff
@@ -186,7 +189,36 @@ func createCmd(c *cli.Context) error {
 	if err != nil {
 		return errors.Wrapf(err, "error creating libpod runtime")
 	}
-	_ = runtime.GetConfig()
 
-	return errors.Errorf("NOT IMPLEMENTED")
+	createConfig, err := parseCreateOpts(c)
+	if err != nil {
+		return err
+	}
+
+	runtimeSpec, err := createConfigToOCISpec(createConfig)
+	if err != nil {
+		return err
+	}
+
+	ctr, err := runtime.NewContainer(runtimeSpec)
+	if err != nil {
+		return err
+	}
+
+	// Should we also call ctr.Create() to make the container in runc?
```

Comment:
I would figure this happens in NewContainer?

Reply:
No, per @mrunalp they're separate operations in the API - NewContainer sets up storage, CreateContainer makes it in the runtime

Reply:
Ok,  Sounds good